### DISCUSSION TITLE
Default App Issue (select the program to open XYZ files.)

### DISCUSSION THREAD

**Discussion by vyrx-dev** [prasanthrangan/hyprdots#1127] [Q&A] [answered]
I've been facing a recurring issue on my system where I'm constantly prompted to choose an application every time I try to open image, video, audio, or PDF files. I've tried setting a default app, but the prompt keeps coming back.  

Any suggestions on how to solve this issue?

![240320_16h32m33s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/139481531/143bbbaf-e19e-439b-aff6-12ead6cd7615)
![240320_16h32m49s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/139481531/3cc74368-3ce4-414c-b6c0-4accf769e68d)

**Comment by kRHYME7** [comment, marked as answer]
Install ``` kservice5 ```

**Reply by elohim-etz** [reply]
After installing kservice5, I'm still experiencing the same issue. What else can I do?


![image](https://github.com/prasanthrangan/hyprdots/assets/133593113/cecd3d92-0617-49d9-9796-f6dab05f0d39)

**Reply by kRHYME7** [reply]
Install something just to refresh the cache like nvim or browser or anything that you expect to be present on that window. Maybe a restart would work.

**Reply by UcraNerd** [reply]
@theamit-969 what did you do to solve the issue?

**Reply by kRHYME7** [reply]
Actually still got this issue after some KDE dependencies update.

**Reply by dieBakterie** [reply]
`sudo downgrade kservice5` to 5.115.0-1 solves this. Logout afterwards should be enough

**Reply by UcraNerd** [reply]
thx! I'm gonna try it right now. :)

**Comment by dieBakterie** [comment]
[How to fix](https://github.com/prasanthrangan/hyprdots/issues/1406)
